wrappers/Python/pytest/test_parity.py

```
@@ -0,0 +1,290 @@
+        except Exception as e:  # noqa: BLE001 -- build may not be supported; not a marshalling bug
+            _assert_not_marshalling_error(e)
+            return
+        for attr in ("tau", "delta", "M1"):
+            assert hasattr(sd, attr), f"SpinodalData missing {attr}"
+
+    def test_extract_backend_returns_pair(self):
+        backend, fluid = C.extract_backend("REFPROP::Water")
+        assert backend == "REFPROP" and fluid == "Water"
+
+    def test_extract_backend_default(self):
+        backend, fluid = C.extract_backend("Water")
```

Ruff (0.15.15)

[warning] Unpacked variable `backend` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)